components/oauth-client/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "oauth-client"
+version = "0.0.0"
+authors = ["Josh Black <raskchanky@gmail.com>"]
```

Comment:
After all the work you've done, I think your name is certainly justified in being here as well 😁

Reply:
:)